iFixit/react-commerce
# Add product page caching

closes #1299 

This PR finally introduces the product model cache 🚀 

## QA

Perform a smoke test on product pages to verify that they still work correctly. If you want to verify that the cache is working you can just check the realtime logs for this deployment on Vercel, and you should see logs about the cache

<img width="1249" alt="Screenshot 2023-01-25 at 18 03 09" src="https://user-images.githubusercontent.com/4640135/214630487-18b6834d-9f35-4d24-9a44-2348a8e12461.png">
